content/en/docs/tutorials/cluster-management/install-use-dra.md

```diff
@@ -0,0 +1,500 @@
+No resources found
+```
+
+#### Check the `ResourceSlices`
+
+The ResourceSlice API resources represent a centralized list of the device
+classes available on different nodes. Initially, there should be no
+ResourceSlices advertised as no DRA drivers have been installed on any nodes yet.
+
+2. Check with `kubectl`:
+
+```shell
+kubectl get resourceslices
+```
+The output is similar to this:
+```
+No resources found
+```
+
+#### Check the `ResourceClaims` and `ResourceClaimTemplates`
+
+ResourceClaims and ResourceClaimTemplates resources encapsulate the requests
+Pods can reference to request or prefer different types of specialized hardware.
+These are further described later, but you can see for now that there are no
+such objects stored yet as you have not created any.
+
+3. Check with `kubectl`
+```shell
+kubectl get resourceclaims
+kubectl get resourceclaimtemplates
+```
+The output is similar to this:
+```
+No resources found
+No resources found
+```
+
+### Checkpoint
+
+At this point, you have confirmed that DRA is enabled and configured properly in
+the cluster, and that no DRA drivers have advertised any resources to the DRA
+APIs yet.
+
+## Install your DRA driver
+
+DRA drivers are third-party applications that run on each node of your cluster
+to interface with the hardware of that node and Kubernetes' native DRA
+components. The installation procedure depends on the driver you choose, but is
+likely deployed as a DaemonSet to all or a selection of the nodes (using node
+selectors or similar mechanisms) in your cluster.
+
+Check your driver's documentation for specific installation instructions, which
+may include a helm chart, a set of manifests, or other deployment tooling.
+
+To illustrate a common installation procedure in this tutorial, you will use an
+example driver which can be found in the
+[kubernetes-sigs/dra-example-driver](https://github.com/kubernetes-sigs/dra-example-driver)
+repository.
+
+### {{% heading "procedure" %}}
+
+#### Download the source code for the example driver
+
+1. Clone the repository:
+
+```shell
+git clone https://github.com/kubernetes-sigs/dra-example-driver.git
+cd dra-example-driver
+```
+
+#### Enable access to your DRA driver's image
+
+In a production environment, you would likely be using a previously released or
+qualified image from the driver vendor or your own organization, and your nodes
+would need to have access to the image registry where the driver image is
+hosted. In this tutorial, you will build an image locally to simulate access to a
+DRA driver image. 
+
+2. Build an image containing the binary for the example driver:
+
+```shell
+./demo/build-driver.sh
+```
+
+#### Deploy the DRA driver components
+
+For this tutorial, you will install the critical example resource driver
+components individually with `kubectl`.
+
+{{< note >}}
+You can deploy all components from this section at once from one file with
+`kubectl apply -f https://k8s.io/examples/dra/driver-install/all.yaml`.
```

Comment:
I have anything namespaced going into the default namespace for ease of use in the tutorial, but is that not recommended? Can change, just lmk.